lib/rust/metamodel/src/java/implementation.rs

```diff
@@ -299,8 +299,14 @@ fn implement_getter(graph: &TypeGraph, class: &Class, id: FieldId) -> syntax::Me
 /// specified [`Field`]. The method must be attached to the same [`syntax::Class`] in which the
 /// [`Field`] is defined.
 fn getter(graph: &TypeGraph, field: &Field) -> syntax::Method {
-    let getter_name = |field| {
-        let field = crate::meta::Identifier::from_camel_case(field);
+    let getter_name = |field: &str| {
```

Comment:
Fix for https://github.com/enso-org/enso/pull/3627/files#r940887460 - internal field names can be prefixed by `field`, but we shouldn't change the externally visible API.